Tests/Marketplace/upload_packs.py

```diff
@@ -787,6 +829,48 @@ def get_packs_summary(packs_list):
     return successful_packs, skipped_packs, failed_packs
 
 
+def are_there_private_packs_to_upload(public_index_folder_path, private_bucket_name, extract_destination_path,
```

Comment:
```suggestion
def handle_private_content(public_index_folder_path, private_bucket_name, extract_destination_path,
```
This function also updates the public index so IMO let's give it a generic name